### ISSUE TITLE
Message to telegram not sent if it exceeds 4096 characters

### ISSUE THREAD

**Issue by alex-voigt** [ix-ai/alertmanager-notifier#6]
Would be really cool to send alerts one by one instead of the summaries like it is done currently (or maybe make it configurable). I was wondering why I got no notifications anymore and it looks like the message that needs be send is just to long:

```
2020-06-18 18:25:20.387 ERROR [telegram_notifier.send] Skipping retries. The exception: Message is too long
2020-06-18 18:26:20.868 INFO [alertmanager-notifier.parse_request] Received 26 alert(s).
2020-06-18 18:26:21.004 ERROR [telegram_notifier.send] Skipping retries. The exception: Message is too long
2020-06-18 18:27:45.038 INFO [alertmanager-notifier.parse_request] Received 26 alert(s).
2020-06-18 18:27:45.171 ERROR [telegram_notifier.send] Skipping retries. The exception: Message is too long
2020-06-18 18:28:32.791 INFO [alertmanager-notifier.parse_request] Received 26 alert(s).
2020-06-18 18:28:32.927 ERROR [telegram_notifier.send] Skipping retries. The exception: Message is too long
```

**Comment by tlex** [comment]
Indeed, I now see that Telegram [has a message size limit of 4096 chars](https://core.telegram.org/method/messages.sendMessage) and alertmanager-notifier doesn't check the size before sending.

I'll take a look at possible solutions.

**Comment by tlex** [comment]
After a bit of digging, sending separate notifications is a no-go as well, since it will run in the [flood protection](https://core.telegram.org/bots/faq#my-bot-is-hitting-limits-how-do-i-avoid-this).

I'll probably need to truncate the message - by removing some of the alerts and putting in a notification about it.

**Comment by tlex** [comment]
I've implemented a different message to be sent to telegram, if the original one reaches 4096 characters.

Basically, you receive a message saying:

> `20 alert(s) received`
> **WARNING!**: The full alert list cannot be sent, since it is larger (**6851** characters) than the maximum message limit for Telegram (**4096** characters). Check the logs for details.

For documentation purpose, the merge request is here: https://gitlab.com/ix.ai/alertmanager-notifier/-/merge_requests/4

I'll merge it during the weekend and create a new release. If you want to take it for a spin before that happens, you can  use the image `ixdotai/alertmanager-notifier:dev-branch`.

**Comment by tlex** [comment]
Closed by e69fde3

Implemented in [v0.1.2](https://github.com/ix-ai/alertmanager-notifier/releases/tag/v0.1.2)

**Comment by alex-voigt** [comment]
Thanks for the fast response! :-) I will check it out in the next days
